### PR TITLE
drivers: wifi: Update RPU patch to rev#a5a1b88e8d4

### PR DESCRIPTION
**Pull request by chiranjeevi2776**
UMAC:
 * Revert intelligent PS patch, this is causing data stall issues on multiple associations

### CODE REVIEW

**Review by krish2718 — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Review by srinivas1201 — APPROVED** [review]

**Comment by krish2718** [comment]
Please wait for approval from @LuDuda to confirm this is solving the matter issue.

**Review by LuDuda — APPROVED** [review]

**Pull request merged**
